package.json

```diff
@@ -84,7 +84,7 @@
     "jest": "^29.7.0",
     "jest-silent-reporter": "^0.5.0",
     "lodash": "^4.17.21",
-    "prettier": "^2.7.1",
+    "prettier": "^3.4.2",
```

Comment:
auto-changelog requires an upgrade of prettier https://github.com/MetaMask/auto-changelog/blob/main/CHANGELOG.md

<img width="803" alt="Screenshot 2024-12-10 at 10 16 02 AM" src="https://github.com/user-attachments/assets/b26d23dc-57d0-4bbd-8409-d930da94acad">